content/docs/installation/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Installation"
+date: 2024-03-03
+weight: 1
+---
+
+The orchestrator solution is made of various independant components where each has its own
```

Comment:
s/independant/independent

Reply:
ahh ok I finally set my vim to detect spelling mistakes. thanks